### ISSUE TITLE
Disable register screen for empty WOOEY_REGISTER_URL

### ISSUE THREAD

**Issue by kdy999** [wooey/Wooey#90]
If  WOOEY_REGISTER_URL is empty then logout now work.
I write litttle patch for urls.py:
```bash
--- /usr/lib/python3.5/site-packages/wooey/urls.py  2015-10-28 09:20:22.326840565 +0300
+++ urls.py 2015-10-28 09:05:52.000000000 +0300
@@ -45,7 +45,6 @@
     url(r'^$', views.WooeyHomeView.as_view(), name='wooey_home'),
     url(r'^$', views.WooeyHomeView.as_view(), name='wooey_job_launcher'),
     url('^{}'.format(wooey_settings.WOOEY_LOGIN_URL.lstrip('/')), views.wooey_login, name='wooey_login'),
-    url('^{}'.format(wooey_settings.WOOEY_REGISTER_URL.lstrip('/')), views.WooeyRegister.as_view(), name='wooey_register'),

     url(r'^favorite/toggle$', views.toggle_favorite, name='toggle_favorite'),

@@ -53,6 +52,9 @@

 ]

+if wooey_settings.WOOEY_REGISTER_URL != "":
+    wooey_patterns += [ url('^{}'.format(wooey_settings.WOOEY_REGISTER_URL.lstrip('/')), views.WooeyRegister.as_view(), name='wooey_register'), ]
+
 urlpatterns = [
     url('^', include(wooey_patterns, namespace='wooey')),
     url('^', include('django.contrib.auth.urls')),
```

**Comment by Chris7** [comment]
Could you submit this as a pull request please?

**Comment by Chris7** [comment]
One problem is we reference wooey:wooey_register in templates. How will this handle that?

**Comment by Chris7** [comment]
Fixed via #143
